src/org/opensolaris/opengrok/web/WebappListener.java

```diff
@@ -99,9 +101,12 @@ public void contextInitialized(final ServletContextEvent servletContextEvent) {
         }
 
         try {
-            RuntimeEnvironment.getInstance().loadStatistics();
+            RuntimeEnvironment runtimeEnvironment = RuntimeEnvironment.getInstance();
+            if (runtimeEnvironment.getConfiguration().getDataRoot() != null) {
```

Comment:
null value check could be done in loadStatistics() method, but breaks StatsMessageTest.java => testInvalidReload, since this test expects exception to be thrown